sandy/core/lpc.py

```diff
@@ -229,7 +229,8 @@ def custom_perturbation(self, mat, mt, p, pert, inplace=False):
         else:
             return Lpc(u_lpc.data)
 
-    def reshape(self, eg, selected_mat=None, selected_mt=None, inplace=False):
+    def reshape(self, eg, selected_mat=None, selected_mt=None, inplace=False,
```

Comment:
can you remove the `inplace` option. to be consistent with other classes I remove `inplace from all methods`